staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy.go

```diff
@@ -154,13 +138,26 @@ func (r *proxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	newReq, cancelFn := newRequestForProxy(location, req)
 	defer cancelFn()
 
-	if handlingInfo.proxyRoundTripper == nil {
+	roundTripper, err := handlingInfo.dynamicRestConfig.GetTransport()
```

Comment:
this is a hot path, now it depends on an external component, for example, it can return an error.

Reply:
any suggestions how to resolve this?